src/core_modules/capture-core/components/DataEntries/Enrollment/actions/enrollment.actionBatchs.js

```diff
@@ -38,7 +39,7 @@ export const runRulesOnUpdateFieldBatch = (
     });
 
     return batchActions([
-        updateRulesEffects(effects, formId),
+        updateRulesEffects(effects, formId, formBuilderId),
```

Comment:
Hey,
Looks like the bug is fixed for the newEnrollment form. In order to fix the same error in the rest of the forms, does it make sense to use `formBuilderId` every time `updateRulesEffects` is called?
Thanks!

Reply:
hey @simonadomnisoru , I'm not entirely sure what do you mean why the rest of the form, can you give an example?

Reply:
Sure, some examples that come to mind from the rest of the forms are: 

**1) Add an event in a tracker program** 
<img width="1035" alt="add new event in tracker programs" src="https://user-images.githubusercontent.com/6817073/214306910-848ed2a5-3705-4b67-9252-679093d65207.png">

**2) Edit an event in a tracker program** 
<img width="1008" alt="edit event in tracker programs" src="https://user-images.githubusercontent.com/6817073/214306945-cf4c1872-f06c-42ac-a67e-1d15a3b2eb5f.png">

**3) Edit the TEI attributes**
<img width="905" alt="edit tei attributes" src="https://user-images.githubusercontent.com/6817073/214307006-54b605d4-a340-44ab-a8d7-8401b3f73e1d.png"> 

**4) Add an event in an event program** 
<img width="951" alt="add new event in event program" src="https://user-images.githubusercontent.com/6817073/214307029-b4e5c77a-f6ae-4309-9817-60ff2e1983ca.png">

It seems that the assigned value is also not treated as an entered value in these scenarios. I believe if `formBuilderId` is used every time `updateRulesEffects` is called, then the error would be fixed in all forms.

Let me know if there are any further doubts. Thanks!

Reply:
hey @simonadomnisoru 
I can not reproduce the problem on the stated scenarios, can you share how you set up the program rules? thanks!

![2](https://user-images.githubusercontent.com/89806888/214341846-960c43da-0b83-498c-b003-0b1a42fb94d3.png)
![1](https://user-images.githubusercontent.com/89806888/214341858-49dda1db-f47f-4854-bb3f-ebbc404e9154.png)
<img width="1020" alt="Screenshot 2023-01-24 at 15 50 22" src="https://user-images.githubusercontent.com/89806888/214341865-8eb003fa-c301-4a39-8205-ca046dc7cd71.png">
<img width="1096" alt="Screenshot 2023-01-24 at 15 50 02" src="https://user-images.githubusercontent.com/89806888/214341869-482ca1fe-208d-4dff-876e-71cb9c46bc72.png">